src/apps/backend/modules/shared-tasks/internal/shared-task-util.ts

```diff
@@ -0,0 +1,14 @@
+import { SharedTask } from '../types';
+import { SharedTaskDB } from './store/shared-task-db';
+
+export default class SharedTaskUtil {
+  public static convertSharedTaskDBToSharedTask(
+    sharedTaskDb: SharedTaskDB
+  ): SharedTask {
+    const sharedTask = new SharedTask();
+    sharedTask.id = sharedTaskDb._id.toString();
+    sharedTask.task = sharedTaskDb.task.toString(),
```

Comment:
Could you handle both task cases? As task can be objected as well as Object< Task >